README.md

```diff
@@ -13,15 +13,20 @@ based services and apps, or run AWS ops tasks.
 
 ## Getting started
 
-See the [Elastic CI Stack for AWS tutorial](https://buildkite.com/docs/guides/elastic-ci-stack-aws) for a step-by-step guide, the [Elastic CI Stack for AWS documentation](https://buildkite.com/docs/agent/v3/elastic-ci-aws), or the full [list of recommended resources](#recommended-reading) for detailed information.
+Learn more about getting started with the Elastic CI Stack for AWS in the Buildkite Docs:
 
-Or jump straight in:
+- [Elastic CI Stack for AWS overview](https://buildkite.com/docs/agent/v3/elastic-ci-aws/elastic-ci-stack-overview) page, which outlines how the Elastic CI Stack works.
+- [Linux and Windows setup for the Elastic CI Stack for AWS](https://buildkite.com/docs/guides/elastic-ci-stack-aws) page for a step-by-step guide on how to set up the Elastic CI Stack in AWS for these operating systems.
+
+A [list of recommended resources](#recommended-reading) provides links to other pages in the Buildkite Docs for more detailed information.
```

Comment:
Btw, as you may have noticed, I'm not a fan of providing links to target pages where the linking text doesn't match the name/title of the target page. Ensuring such text matches provides a better reading experience, and reduces ambiguity. For example, 'tutorial' (in this context) refers to the [Linux and Windows setup for the Elastic CI Stack for AWS](https://buildkite.com/docs/agent/v3/elastic-ci-aws) page, but what about the [EC2 Mac setup for the Elastic CI Stack for AWS](https://buildkite.com/docs/agent/v3/elastic-ci-stack-for-ec2-mac/autoscaling-mac-metal) page? Is that not a tutorial too?

Reply:
That's a very good idea and a great catch to link to the Mac setup for the Elastic CI Stack, @gilesgas! Please add it.